code/go/0chain.net/smartcontract/benchmark/main/cmd/bls0chain.go

```diff
@@ -0,0 +1,274 @@
+package cmd
+
```

Comment:
Any reason for why we don't use the `encryption/bls0chain.go` directly, and do a copy here.